shell.nix

```
@@ -498,7 +498,7 @@ All binaries are for 64-bit operating systems.
    # gets a markdown version of pulse
    # greps for everything from summary to details (not including details heading)
    # deletes null characters that throw warnings in bash
-   PULSE_NOTES=$( curl -s https://md.unmediumed.com/${pulse-url} | grep -Pzo "(?s)(###.*Summary.*)(?=###.*Details)" | tr -d '\0' )
+   PULSE_NOTES=$( curl -s https://md.unmediumed.com/${pulse-url} | grep -Pzo "(?s)(###.*Summary.*)(?=###\s+\**Details)" | tr -d '\0' )
```

```suggestion
   PULSE_NOTES=$( curl -s https://md.unmediumed.com/${pulse-url} | grep -Pzo "(?s)(###\s+\**Summary.*)(?=###\s+\**Details)" | tr -d '\0' )
```

@thedavidmeister three thoughts:

1. it seems good to me to apply the same fix to the 'Summary' header as well. (See suggested change)
2. re: `\**` what is this fixing? Is the 'Details' element sometimes wrapped in bold or italic, _in addition to_ being a header? (I can totally believe it; just making sure I understood that piece of regex.)
3. Might be good to future-proof the `###` against the styling idiosyncrasies of future social media managers by going `#{1,}` instead. May be paranoid, but give the way that people use WYSIWYG editors, I reckon paranoia is entirely justified :D

---

oh blast -- I didn't refresh this page and discover it'd already been merged :grimacing: whoops.